docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md

```diff
@@ -0,0 +1,111 @@
+
+**Keep selectors up-to-date:** Regularly review and update your selectors to ensure they are still accurate and efficient. When the application under test changes, be proactive in updating the selectors in your page files to prevent your tests from breaking
+
+
+## Use of annotations for skipping tests with Native Playwright
+
+This relates to working on these repo's:
+- Eden (DAM E2E tests)
+- Qts-Clients (Cliend Manager E2E Tests)
+- Qtui-front-end (Front-End E2E Tests)
+
+When maintaining tests, it is important to use the correct annotations to handle test logic effectively as required, ie, skipping tests.  This is to ensure that the entire test suite is not unnecessarily run and to prevent false negatives. Integrating annotations into our Playwright workflow can dramatically improve our testing efficiency and enable us to gain precise control over which tests run and how they behave.
+
+For whichever annotation we use, it is imperative that we put a comment above the annotation to justify why it's being used, to ensure it's clear to other team members.
+
+For further examples of annotations playwright support, please follow the Playwright documentation found [here](https://playwright.dev/docs/test-annotations)
+
+**In summary:**
+
+- Use `test.only` to run a single test. (Playwright will only run the test with the `test.only` annotation applied to it).
+
+This should be used in instances where we want to focus on a single test and ignore all other tests. 
+
+**We should **NEVER** merge tests with `test.only` annotations into the main branch, but it has it's uses when executing tests locally or on work / feature branches at times.**
+
+- Use `test.fixme` to mark a test that is known to be failing and requires a fix. (Playwright will not run the test and it will be skipped).
+
+This should be used in instances where the test logic is fine but the test is failing due to a known defect. There should be a BLI in place to fix this within the next 2 sprints and we are aware a fix is in the works. Its imperative we include a comment above the `test.fixme` annotation to explain why the test is failing, along with a reference ID of the DevOps BLI. 
+
+**We can merge tests with `test.fixme` annotations into the main branch, but only temporarily. This should then be reverted back to `test` once the test is fixed.**
+
+- Use `test.skip` to skip a test. (Playwright will not run the test and it will be skipped).
+
+There are 2 main instances why we would use this annotation:
+
+1 - To skip a test temporarily in order to focus on other tests. This can be beneficial when running tests *locally* or on work / feature branches, but shouldnt be merged to main in these instances. 
+
+2 - To skip a work in progress test, whereby the logic in the test would fail at this particular instance in time because it's not ready for testing yet, but we want to keep the test in the codebase for future reference or enhancement.
```

Comment:
Should we mention this is rare as we wouldn't want to merge incomplete work as an every day practice?

Reply:
updated :)